app/assets/javascripts/activities.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function(){
+	$('#activity_start_time, #activity_end_time').timepicker({
```

Comment:
What indentation style are you using for JS?
Rails developers generally use 2 space indentation for all files. I find using 2 space indentation works good, especially for JS, as JS code generally has multiple levels of indentation.